istio/metadata.csv

```diff
@@ -1,327 +1,356 @@
-istio.go.memstats.heap_idle_bytes,gauge,,byte,,[Istio v1.5+] Number of idle bytes in the heap.,0,istio,
-istio.go.memstats.heap_inuse_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes in the heap.,0,istio,
-istio.go.memstats.heap_objects,gauge,,object,,[Istio v1.5+] Number of objects in the heap.,0,istio,
-istio.go.memstats.heap_released_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes released to the system in the last gc.,0,istio,
-istio.go.memstats.heap_sys_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes used by the heap.,0,istio,
-istio.go.memstats.last_gc_time_seconds,gauge,,second,,[Istio v1.5+] Length of last GC.,0,istio,
-istio.go.memstats.lookups_total,count,,operation,,[Istio v1.5+] Number of lookups.,0,istio,
-istio.go.memstats.mallocs_total,count,,operation,,[Istio v1.5+] Number of mallocs,0,istio,
-istio.go.memstats.mcache_inuse_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes in use by mcache structures.,0,istio,
-istio.go.memstats.mcache_sys_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes used for mcache structures obtained from system.,0,istio,
-istio.go.memstats.mspan_inuse_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes in use by mspan structures.,0,istio,
-istio.go.memstats.mspan_sys_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes used for mspan structures obtained from system.,0,istio,
-istio.go.memstats.next_gc_bytes,gauge,,byte,,[Istio v1.5+] Number of heap bytes when next garbage collection will take place.,0,istio,
-istio.go.memstats.other_sys_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes used for other system allocations.,0,istio,
-istio.go.memstats.stack_inuse_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes in use by the stack allocator.,0,istio,
-istio.go.memstats.stack_sys_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes obtained from system for stack allocator.,0,istio,
-istio.go.memstats.sys_bytes,gauge,,byte,,[Istio v1.5+] Number of bytes obtained from system.,0,istio,
-istio.go.threads,gauge,,thread,,[Istio v1.5+] Number of OS threads created.,0,istio,
-istio.process.cpu_seconds_total,count,,second,,[Istio v1.5+] Total user and system CPU time spent.,0,istio,
-istio.process.max_fds,gauge,,file,,[Istio v1.5+] Maximum number of open file descriptors.,0,istio,
-istio.process.open_fds,gauge,,file,,[Istio v1.5+] Number of open file descriptors.,0,istio,
-istio.process.resident_memory_bytes,gauge,,byte,,[Istio v1.5+] Resident memory size in bytes.,0,istio,
-istio.process.start_time_seconds,gauge,,second,,[Istio v1.5+] Start time of the process since unix epoch in seconds.,0,istio,
-istio.process.virtual_memory_bytes,gauge,,byte,,[Istio v1.5+] Amount of virtual memory used.,0,istio,
-istio.process.virtual_memory_max_bytes,gauge,,byte,,[Istio v1.5+] Maximum amount of virtual memory available.,0,istio,
+istio.mesh.request.count,count,,request,,"[OpenMetrics V1 and V2 and Istio v1.1+] The number of requests. This metric is sent as gauge by default in OpenMetrics V1.",0,istio,request count
```

Comment:
we can drop v1.1+, the only version that matters it `1.5+` because of changes to Istio architecture and the deprecation of mixer/pilot/gallet/citadel endpoints in favor or istiod_endpoint.